core/base/common/ArrayLinkedList.h

```diff
@@ -0,0 +1,37 @@
+/// \ingroup base
+/// \class ttk::ArrayLinkedList
+/// \author Eve Le Guillou <eve.le-guillou@lip6.fr>
+/// \date Mars 2022.
+
+#include <array>
+#include <list>
+
+#pragma once
+
+namespace ttk {
+  template <typename datatype, int size>
+  class ArrayLinkedList {
+  public:
+    std::list<std::array<datatype, size>> list;
+    int numberOfElement;
```

Comment:
please use the prefix `_` for all member variables.
`list` -> `list_`
`numberOfElement` -> `numberOfElements_` (plural)
`padding` -> `padding_`